census/controllers/census.js

```diff
@@ -290,15 +290,17 @@ var pending = function(req, res) {
         reviewComments: entry.reviewComments
       };
 
+      let reviewersData = {place: place, dataset: dataset};
```

Comment:
I usually go with `const` for things that aren't likely to / should not change after declaring. But that's entirely up to you.